execution/gethexec/forwarder.go

```diff
@@ -116,34 +121,45 @@ func NewForwarder(target string, config *ForwarderConfig) *TxForwarder {
 		ExpectContinueTimeout: 1 * time.Second,
 	}
 	return &TxForwarder{
-		target:    target,
-		timeout:   config.ConnectionTimeout,
-		transport: transport,
+		targets:               targets,
+		timeout:               config.ConnectionTimeout,
+		transport:             transport,
+		tryNewForwarderErrors: regexp.MustCompile(`(?i)(http:|json:|.*Timeout exceeded).*`),
```

Comment:
We should also include whatever error happens if DNS resolution fails because the domain doesn't exist, and `i/o.*`